dotCMS/src/integration-test/java/com/dotmarketing/portlets/htmlpages/business/render/HTMLPageAssetRenderedAPIImplIntegrationTest.java

```diff
@@ -2243,17 +2256,6 @@ public void usingViewToolToInjectExperimentCode() throws IOException {
         }
     }
 
-    /**
-     * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
-     * When: You don't have any {@link Experiment} RUNNING but the Experiment and Auto Injection are ENABLED
-     * but the Page is not a HTML code.
-     * Should: not inject any code
-     */
-    @Test
-    public void experimentAndInjectionEnabledButNotHTMLPage(){
-        throw new RuntimeException();
-    }
-
     private String getNotExperimentJsCode(){
         return "<SCRIPT>localStorage.removeItem('experiment_data');</SCRIPT>\n";
```

Comment:
This will be good to centralize it as mentioned in https://github.com/dotCMS/core/pull/24321/files#r1137237673

Reply:
This is a test so I think it should go hardcoding

Reply:
yeah hardcoding is better for this case